modules/rosa-upgrading-automatic-ocm.adoc

```diff
@@ -5,16 +5,18 @@
 
 :_content-type: PROCEDURE
 [id="rosa-scheduling-upgrade_{context}"]
-= Scheduling automatic upgrades
+= Scheduling recurring upgrades for your cluster
 
-You can schedule automatic upgrades for a {product-title} cluster through {cluster-manager-first} console.
+You can schedule recurring, automatic upgrades for your {product-title} cluster on a weekly basis through the {cluster-manager} console.
 
 .Procedure
 
 . Log in to {cluster-manager-url}.
 . Select a cluster to upgrade.
 . Click the *Settings* tab.
-. In the *Update strategy* pane, click *Automatic* and select a preferred day of the week and start time for the automatic upgrades.
+. In the *Update strategy* pane, select *Recurring updates*.
+. Select a preferred day of the week and start time for the upgrade, when updates are available.
+. Provide an administrator’s acknowledgment and click *Approve and continue*. {cluster-manager} does not start scheduled y-stream updates for minor versions without receiving an administrator’s acknowledgment.
```

Comment:
Is there a difference between _upgrade_ and _update_?
start time for the _upgrade_
when _updates_ are available
scheduled y-stream _updates_

Reply:
This requires a larger conversation with the PM team and the decision will impact more topics than the ones touched on in this PR. I'll create a separate PR to track the update vs upgrade term change, but meanwhile, this PR needs to be merged. Thank you.